src/js/components/__tests__/__snapshots__/components-test.js.snap

```diff
@@ -1247,6 +1247,15 @@ exports[`Components loads 1`] = `
           "margin": {
             "bottom": "small",
           },
+          "survey": {
+            "label": {
+              "margin": {
+                "vertical": "xsmall",
```

Comment:
The spacing between question and input and also between formfields looks off -- would be good to double check the desired value.

Reply:
I think we should include "survey" in the base grommet theme. It's a reasonable concept that I think would be good to provide.